### PR TITLE
Feature/bound players slack

### PR DESCRIPTION
**Pull request by eberbis**
## GOAL
To post on Slack the bound players (top and bottom players of a league) once a match has being played. 
This way they will know how they are performing without going to the actual page.

## APPROACH
* Create a new function that will post to slack these values.
* Add that function to the slack loop.
* Make a test for it.

## ADDITIONAL COMMENTS
* This is an ongoing feature. 
* This PR has been made to allow more fluent Clojure speakers to help on the task.
* All feedback is much appreciated.

### CODE REVIEW

File: src/clj/react_tutorial_om/events.clj [hunk 1]
```diff
@@ -21,12 +21,35 @@
...
+          (println (:throwable &throw-context) "unexpected error")))))
+
+
 (defn setup-slack-loop [channel prefix slack-url]
   (go-loop []
     (when-let [[topic msg] (<! channel)]
       (println prefix ": " topic ": " msg)
       (case topic
         :league-match (post-league-result-to-slack msg slack-url)
+        :bound-players (post-league-bound-players-to-slack msg slack-url)
```
Comment (minimal) [inline src/clj/react_tutorial_om/events.clj]: i think the league match event should also post the top/bottom, currently it will never get triggered

Comment (eberbis) [inline src/clj/react_tutorial_om/events.clj]: OK, it's a good idea. 
The tests would need to be updated to englobe that post to slack and we need to pass the top 2 players and bottom 2 players to that function. 
Not sure where that info is passed (`msg`?)

Comment (minimal) [inline src/clj/react_tutorial_om/events.clj]: I think if we pass a snapshot of the "db" atom (or just the league part) then an event handler will have all the info it needs

e.g.

```clj
(defn handle-league-result
  "Given an db atom, a league name and a result update the schedule and
  matches for the league and write out to file. Return the resulting
  state. Also post to slack if possible."
  [db league {:keys [winner loser winner-score loser-score] :as result} event-ch]
  (let [db-snapshot (swap! db (partial update-league-result result league))]
    (go (>! event-ch [:league-match (assoc result
                                           :league league
                                           :league-snapshot (get-in db-snapshot [:leagues league]))])))
  {:message "ok"})
```

**Comment by minimal** [comment]
Incorporated the string formatting and modified a bit. It now lives here https://github.com/minimal/player-ladder/blob/f0fc656a0ec786c94f9f05ec4e85921f0b64424b/src/clj/react_tutorial_om/events.clj#L10-L18

**Pull request merged**